widget-context.php

```diff
@@ -418,16 +418,10 @@ function context_check_url( $check, $settings ) {
 
 	function match_path( $patterns ) {
 
-		global $wp;
-
 		$patterns_safe = array();
 
-		// Get the request URI from WP
-		$url_request = $wp->request;
-
-		// Append the query string
-		if ( ! empty( $_SERVER['QUERY_STRING'] ) )
-			$url_request .= '?' . $_SERVER['QUERY_STRING'];
+		// Get the request URI
+		$url_request = trim( $_SERVER['REQUEST_URI'], '/' );

```

Comment:
Yes, it would be relative to root. I guess we should slice off the WP base path first? Assuming WP is at root, it would be an empty string?